fortytwo.cabal

```diff
@@ -35,8 +35,8 @@ library
                        FortyTwo.Renderers.Select
                        FortyTwo.Renderers.Multiselect
   build-depends:       base >= 4.7 && < 5,
-                       text <= 1.3,
-                       ansi-terminal >= 0.6.0.0 && <= 0.12
+                       text < 2.1,
```

Comment:
Please avoid updating the dependencies boundaries with this PR

Reply:
Reverted